### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by Poofless321** [Poofless321/chan-Sankaku-adblock#1] [General]
potato

**Comment by Xzentrick** [comment]
Greetings. You recently updated the chan-Sankaku-adblock. Would you be willing to create a variant of it for " https://beta . sankakucomplex . com/companions?tags=threshold%3A0 " so that the adblock works with the beta portion of the Sankaku website? Sorry for the trouble, as well as the random request. Thank you for your time and any help you may be willing to provide. This is what the intrusive pop-up message that prevents users from doing anything looks like:

![Sankaku](https://github.com/Poofless321/chan-Sankaku-adblock/assets/166598739/c1a38dc8-ca86-4df1-8b64-f3f36c7f8caf)
